test/unit/test_diffmaps.py

```diff
@@ -0,0 +1,217 @@
+import numpy as np
+import pandas as pd
+import pytest
+import reciprocalspaceship as rs
+
+from meteor.diffmaps import compute_deltafofo, compute_kweighted_deltafofo
+
+
+@pytest.fixture
+def sample_dataset() -> rs.DataSet:
```

Comment:
can we combine this / reuse something else in the tests? look in the two `conftest.py` files. Fixtures in those files are automatically usable by all tests. And the code in those files can be improved tbh.